src/components/CollapsedOpeningTimes.js

```diff
@@ -11,6 +11,7 @@ import Collapser from "./general/Collapser";
 import OpeningTimes from "./OpeningTimes";
 import type {openingHours as openingHoursType} from "../iss/general";
 
+
```

Comment:
Is this extra line break required?  Cant remember if linting needs two linebreaks after the imports